### PR TITLE
Several pages on testing considerations

### PR DESCRIPTION
**Pull request by whisk3ykilo**
These pages provide helpful information hacker-powered programs often need but don't think to ask

### CODE REVIEW

File: docs/programs/authenticated-testing.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Authenticated Testing"
+path: "/programs/authenticated-testing.html"
+id: "programs/authenticated-testing"
+---
+
+* HackerOne recommends providing credentials and contextual information to hackers wherever possible
+    * HackerOne recommends offering elevated rewards for unauthenticated vulnerability findings
+* The HackerOne platform includes a secure [credential management feature](https://docs.hackerone.com/programs/credential-management.html) that allows customers to quickly upload many sets of credentials
```
Comment (jessiwright) [inline docs/programs/authenticated-testing.md:9]: "many sets of credentials" might be better as "multiple sets of credentials"

File: docs/programs/traffic-identification.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Traffic Identification"
+path: "/programs/traffic-identification.html"
+id: "programs/traffic-identification"
+---
+
+There are several ways to identify hacker testing traffic at various layers for testing/feature enablement or testing control & monitoring.
+
+### Application Layer: User Allowlisting
+* HackerOne provides each hacker with a forwarding email [address](https://docs.hackerone.com/hackers/hacker-email-alias.html)
```
Comment (jessiwright) [inline docs/programs/traffic-identification.md:10]: Should "email" maybe be part of the link? It seems unusual to me to have "address" linked when it's actually one unit - "email address".

File: docs/programs/authenticated-testing.md (hunk 1 above)
Comment (whisk3ykilo) [reply docs/programs/authenticated-testing.md:9]: Agreed

File: docs/programs/traffic-identification.md (hunk 2 above)
Comment (whisk3ykilo) [reply docs/programs/traffic-identification.md:10]: Yep that's fair

File: docs/programs/authenticated-testing.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Authenticated Testing"
```
Comment (jessiwright) [inline docs/programs/authenticated-testing.md:2]: Should the headers be capitalized here or are they supposed to be lowercase?

**Pull request merged**